pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -2,26 +2,28 @@
 
 ### Deposit to Subaccount
 
+The below cmd allows depositing funds into a subaccount from a sender. This can be a self deposit with the sender and recipient address being the same
 ```bash
 dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
 ```
 
 Example:
 ```bash
 NATIVE_TOKEN_DENOM=adv4tnt
```

Comment:
lets remove this line

Reply:
responded to this on the old PR: [The NATIVE_TOKEN_DENOM var is just for ease of use on the cmd here. I already have the note pointing to the url](https://github.com/dydxprotocol/v4-documentation/pull/110#discussion_r1462077017)

@vincentwschau also suggested the same

Reply:
removed